docs/CMakeLists.txt

```diff
@@ -3,6 +3,6 @@ project(docs)
 
 include(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/ClioVersion.cmake)
```

Comment:
This file and file below were wrongfully not formatted by cmake-format, because it only formatted `cmake` folder